.drone/drone.jsonnet

```diff
@@ -917,6 +916,7 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
           DOCKER_PASSWORD: { from_secret: docker_password_secret.name },
         },
         commands: [
+	  'git fetch origin --tags',
```

Comment:
tried one more approach before this.

Tried to add it as separate step say `fetch-git-tags` on this pipeline. In that case, git tags only fetched in that particular step and cannot be used during `make docker-driver-push` step. Rationale being other pipelines that do use separate step to fetch git tags, stores it in `./tools/image-tag > arch.tags` and re-use it other step.

Ended up doing this way without changing much of the pipelines.